### PR TITLE
Feature/m0078 ddg 409 refactor move platform architecture packs

### PR DESCRIPTION
**Pull request by frivieremicroej**

### CODE REVIEW

**Comment by frivieremicroej** [comment]
https://docs.microej.com/en/feature-m0078ddg-409_refactor_move_platform_architecture_packs/PlatformDeveloperGuide/

File: PlatformDeveloperGuide/architecture.rst [hunk 1]
```diff
@@ -0,0 +1,64 @@
+.. _architecture_overview:
+
+====================
+MicroEJ Architecture
+====================
+
+Introduction
```
Comment (gjadi-mej) [inline PlatformDeveloperGuide/architecture.rst:7]:
```suggestion
Overview
```

I'd say it's more an overview than an introduction to MicroEJ Architecture.  To me, `introduction` means that we are about to see something /after/ the introduction section.  But that's not the case here, the next section is on the Naming Convention.

File: PlatformDeveloperGuide/pack.rst [hunk 2]
```diff
@@ -0,0 +1,94 @@
+.. _pack_overview:
+
+=============
+MicroEJ Packs
+=============
+
+Introduction
```
Comment (gjadi-mej) [inline PlatformDeveloperGuide/pack.rst:7]:
```suggestion
Overview
```

Same argument as for MicroEJ Architecture.

File: PlatformDeveloperGuide/platformOverview.rst [hunk 3]
```diff
@@ -1,64 +1,34 @@
...
-See :ref:`MicroEJ Architecture Import <architecture_import>` for usage.
-
-.. _pack_overview:
-
-MicroEJ Packs
-~~~~~~~~~~~~~
+-  Documentation: this guide, library specifications, etc.
 
-On top of a MicroEJ Architecture can be imported MicroEJ Packs which provide additional features such as:
+-  Tools for development and compilation.
```
Comment (gjadi-mej) [inline PlatformDeveloperGuide/platformOverview.rst:22]:
```suggestion
-  Tools for development, compilation and validation.
```

**Review by gjadi-mej — COMMENTED** [review]
Some small suggestions.

File: PlatformDeveloperGuide/platformOverview.rst (hunk 3 above)
Comment (frivieremicroej) [reply PlatformDeveloperGuide/platformOverview.rst:22]: This content is outa little bit out-of-date.
Most of MicroEJ Packs no more include tools (except UI Pack generators)
Let's update it after this refactor.

**Pull request merged**